### ISSUE TITLE
setup XDEBUG3 in Windows failed

### ISSUE THREAD

**Issue by jkowalleck** [shivammathur/setup-php#700]
**Describe the bug**
on windows-latest, setup of `xdebug3` fails in latest php8.2

**Version**
- [x] I have checked releases, and the bug exists in the latest patch version of `v1` or `v2`.
- [x] `v2`
- [ ] `v1`

**Runners**

- [x] GitHub Hosted
- [ ] Self Hosted

**Operating systems**

windows-latest

**PHP versions**

8.2.1

**To Reproduce**

```yml
- uses: shivammathur/setup-php@v2
  with:
    php-version: '8.2'
    extensions: dom,json,libxml,simplexml
    tools: composer:v2
    coverage: xdebug
```

**Expected behavior**

`xdebug3` is installed and configured, as described in the docs of this GH action

**Screenshots/Logs**

https://github.com/CycloneDX/cyclonedx-php-library/actions/runs/4115916576/jobs/7105282080
```text
Run shivammathur/setup-php@v2
  with:
    php-version: 8.2
    extensions: dom,filter,json,libxml,simplexml
    tools: composer:v2
    coverage: xdebug
    ini-file: production
  env:
    PHP_VERSION_LATEST: 8.2
    PHP_PROJECT_EXT: dom,filter,json,libxml,simplexml
    REPORTS_DIR: CI_reports
    REPORTS_ARTIFACT: tests-reports
"C:\Program Files\PowerShell\7\pwsh.exe" D:\a\_actions\shivammathur\setup-php\v2\src\scripts\run.ps1

==> Setup PhpManager 
√ PhpManager Installed 

==> Setup PHP 
✗ PHP Could not fetch CA certificate bundle from Curl 
√ PHP Found PHP 8.2.1 

==> Setup Extensions 
√ dom Enabled 
√ filter Enabled 
√ json Enabled 
√ libxml Enabled 
√ simplexml Enabled 

==> Setup Tools 
√ composer Added composer 2.5.2 

==> Setup Coverage 
✗ xdebug Could not install xdebug on PHP 8.2.1 

==> #StandWithUkraine 
√ read-more https://setup-php.com/support-ukraine 
```

**Additional context**

**Are you willing to submit a PR?**

**Comment by jkowalleck** [comment]
update: 
* it appears to be very unstable: Each 3rd try, the setup of `xdebug` works as expected.
* same with `pcov`: each 3rd try, the setup of `xdebug` works as expected.

**Comment by shivammathur** [comment]
Fetching builds on Windows has been unstable recently. I have a patch in the `develop` branch to retry downloading on failure, that should help. it will be in the next release.
This has been reported to GitHub Actions team here https://github.com/actions/runner-images/issues/7007

**Comment by jkowalleck** [comment]
Thank you very much.
Your work is appreciated.

**Comment by shivammathur** [comment]
Fixed in https://github.com/shivammathur/setup-php/commit/f9390a547837c6e73567bcb2599502afda1e0c39

**Comment by codemasher** [comment]
Just came here to report the same issue. Thank you for the quick fix!

**Comment by shivammathur** [comment]
Should be fixed by [`2.25.0`](https://github.com/shivammathur/setup-php/releases/tag/2.25.0) release.
